index.html

```diff
@@ -1,114 +1,147 @@
+            alt="인기 상품 확인"
+            class="feature-image"
+          />
+        </div>
       </section>
-      <section>
-        <img
-          src="assets/landing-page/main-3.png"
-          alt="img-hot-item"
-          class="img-hot-item"
-        />
-        <article class="main-content-text">
-          <h2 class="category hot-item">Hot item</h2>
-          <h1 class="title hot-item">
-            인기 상품을<br />
-            확인해보세요
-          </h1>
-          <p class="description hot-item">
-            가장 HOT한 중고거래 물품을<br />
-            판다 마켓에서 확인해 보세요
+      <section class="feature-container">
+        <div>
+          <img
+            src="assets/feature1.png"
+            alt="인기 상품 확인"
+            class="feature-image"
+          />
+        </div>
+        <div class="feature-text-container">
+          <p class="feature-category">Register</p>
+          <h1 class="feature-title">판매를 원하는<br />상품을 등록하세요</h1>
+          <p class="feature-description">
+            어떤 물건이든 판매하고 싶은 상품을<br />쉽게 등록하세요
           </p>
-        </article>
+        </div>
       </section>
     </main>
     <footer>
-      <section class="banner-section">
-        <div class="banner-content">
-          <p class="banner-text">
-            믿을 수 있는<br />
-            판다마켓 중고 거래
-          </p>
+      <section class="hero">
+        <div class="hero-container">
+          <div class="hero-text-container">
+            <h1 class="hero-title">믿을 수 있는<br />판다마켓 중고 거래</h1>
+          </div>
+          <div class="hero-image-container">
+            <img
+              src="assets/footer-hero.png"
+              alt="판다마켓 푸터 히어로 이미지"
+              class="hero-image"
+            />
+          </div>
         </div>
-        <img src="assets/landing-page/footer.png" alt="footer-image" />
       </section>
-      <section class="footer-section">
-        <div class="footer-copyright">©codeit - 2024</div>
-        <div class="footer-links">
-          <a href="/privacy">Privacy Policy</a>
-          <a href="/faq">FAQ</a>
-        </div>
-        <div class="footer-social">
-          <a href="https://www.facebook.com"
-            ><i class="fa-brands fa-facebook"></i
-          ></a>
-          <a href="https://www.x.com"><i class="fa-brands fa-twitter"></i></a>
-          <a href="https://www.youtube.com"
-            ><i class="fa-brands fa-youtube"></i
-          ></a>
-          <a href="https://www.instagram.com"
-            ><i class="fa-brands fa-instagram"></i
-          ></a>
+      <section class="footer-container">
+        <div class="footer-content">
+          <div class="footer-copyright copyright">
+            <p>©codeit - 2024</p>
+          </div>
+          <div class="footer-links">
+            <a href="/privacy.html">Privacy Policy</a>
+            <a href="/faq.html">FAQ</a>
+          </div>
+          <div>
+            <ul class="footer-social">
+              <li class="social-icons">
+                <a href="#">
```

Comment:
❗️ 수정요청
구조를 변경하시면서 link가 사라졌나봐요~
가능하면 link를 추가해주세요~
```suggestion
                <a href="https://www.facebook.com">
```